### PR TITLE
MM-17294 - Revert "MM-15452 - Add ability to override LHS icon… (#3311)

### PR DESCRIPTION
**Pull request by ali-farooq0**
Cherry pick of #3311 on release-5.14.

- #3311: MM-17294 - Revert "MM-15452 - Add ability to override LHS icon for bot accounts

/cc  @ali-farooq0

### CODE REVIEW

**Comment by ali-farooq0** [comment]
Adding @lieut-data for another set of eyes as I had to manually resolve conflicts for `5.14`

File: package-lock.json [hunk 1]
```diff
@@ -15727,7 +15719,8 @@
     "wrappy": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
-      "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8="
+      "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
+      "dev": true
```
Comment (lieut-data) [inline package-lock.json]: Any idea if this `"dev": true` is intentional?

Comment (ali-farooq0) [reply package-lock.json]: yep it was as part of my inline change in the webapp
https://github.com/mattermost/mattermost-webapp/pull/3026/files#diff-32607347f8126e6534ebc7ebaec4853dL15715

**Comment by ali-farooq0** [comment]
@lieut-data any objections in merging this in?

**Pull request merged**